External/CoordGen/CoordGen.h

```diff
@@ -154,17 +154,19 @@ unsigned int addCoords(T &mol, const CoordGenParams *params = nullptr) {
     auto obnd = *bndit;
     if (obnd->getBondType() != Bond::DOUBLE ||
         obnd->getStereo() <= Bond::STEREOANY ||
-        obnd->getStereo() > Bond::STEREOTRANS)
+        obnd->getStereo() > Bond::STEREOTRANS ||
+        obnd->getStereoAtoms().size() < 2) {
       continue;
+    }
 
     sketcherMinimizerBondStereoInfo sinfo;
-    sinfo.atom1 = ats[obnd->getStereoAtoms()[0]];
-    sinfo.atom2 = ats[obnd->getStereoAtoms()[1]];
+    sinfo.atom1 = ats.at(obnd->getStereoAtoms().at(0));
+    sinfo.atom2 = ats.at(obnd->getStereoAtoms().at(1));
```

Comment:
The inner `.at`s here are not needed since you checked the size above:
```suggestion
    sinfo.atom1 = ats.at(obnd->getStereoAtoms()[0]);
    sinfo.atom2 = ats.at(obnd->getStereoAtoms()[1]);
```

Reply:
I'm not enamored of the other uses of `.at` here or below (we don't generally use this), but I'm not going to argue about it

Reply:
I have reverted all `std::vector::at` back to `std::vector::operator[]`, as I admit they are exceedingly paranoid. I have also checked that `Bond::setStereoAtoms` enforces bounds check on setting, so no need to do an extra bound check.